### DISCUSSION TITLE
Gadget update

### DISCUSSION THREAD

**Discussion by veermaharaj** [LibreHardwareMonitor/LibreHardwareMonitor#656] [Ideas]
Visually, the gadget that is currently available is stuck with a Windows Vista / Win 7 skin. Would there be any interest in updating this with a windows 11 style skin for future-proofing and possibly a windows 10 themed skin so it looks nice with windows 10?

I would be volunteering to do the artwork for the gadgets if need be. I am reasonably competent with photoshop.

**Comment by darkbrain-fc** [comment]
Maybe you can use this: https://github.com/darkbrain-fc/HardwareSupervisorRainmeterPlugin Now latest HardwareSupervisor https://github.com/darkbrain-fc/HardwareSupervisor has migrated to LibreHardwareMonitor

**Comment by edcase7** [comment]
Personally I'd like to be able to turn off the background altogether so that It's transparent... The Gadget is useful & functional, it just looks a bit naff. Additionally Rather than "Font Colours", it would be nice to enter RGB codes to get what you want.
